account-management-app/src/main/java/org/duracloud/account/app/controller/DuracloudMillForm.java

```diff
@@ -84,4 +101,107 @@ public void setAuditLogSpaceId(String auditLogSpaceId) {
         this.auditLogSpaceId = auditLogSpaceId;
     }
 
+    public String getAuditQueueType() {
+        return auditQueueType;
+    }
+
+    public void setAuditQueueType(String auditQueueType) {
+        this.auditQueueType = auditQueueType;
+    }
+
+    public String getRabbitmqHost() {
+        return rabbitmqHost;
+    }
+
+    public void setRabbitmqHost(String rabbitmqHost) {
+        if (settingRabbitMQ()) {
+            if (rabbitmqHost == null) {
+                throw nullConstraintViolationException();
```

Comment:
@bbranan do you think we still need all of these server-side validations now that we're doing client-side validation?

Reply:
Let's leave them in place. It doesn't hurt to double check.